presto-docs/src/main/sphinx/release/release-0.245.rst

```diff
@@ -0,0 +1,34 @@
+=============
+Release 0.245
+=============
+
+**Highlights**
+==============
+* New `ENUM_KEY` UDF to get the key for an enum value.
+* Configurable query result compression.
+
+**Details**
+===========
+
+General Changes
+_______________
+* Fix wrong results issue for queries with multiple lambda expressions that differ only in array/map/row constants that have the same length (:issue:`15424`).
+* Fix enum deserialization bug in map keys.
+* Improve memory usage for fragment result caching.
+* Add support for disabling query result compression via client and server-side configuration properties. Clients can disable compressed responses using the ``--disable-compression`` flag or ``disableCompression`` driver property. Compression can be disabled server-wide by using the configuration property: ``query-results.compression-enabled=false``
+* Add support in affinity scheduler to enable cache for bucketed table scan that has remote source.
+* Add a UDF to get the key corresponding to an enum value: `ENUM_KEY(EnumType) -> VARCHAR`
+* Add SqlParserOptions to prepared statement handler so that parsing logic is consistent with regular requests.
```

Comment:
`Fix parsing logic for prepared statements to be consistent with regular statements` - I don't think we need to outline the method that we achieved this.